packages/svelte2tsx/index.d.ts

```diff
@@ -3,11 +3,24 @@ export interface SvelteCompiledToTsx {
     map: import("magic-string").SourceMap;
     exportedNames: IExportedNames;
     events: ComponentEvents;
+    componentDocumentation: string;
+    slots: Map<string, Map<string, string>>;
+    generics: {
+        definitions: string[];
```

Comment:
The types say these are strings, but that's not true from the implementation (besides it's `definition` in the implementation). In the implementation it's just one string fitting the use case which is producing correct code, not something for use later.